internal/console/helpers/helpers.go

```diff
@@ -109,18 +129,34 @@ func PrintResult(summary *model.Summary, failedQueries map[string]error) error {
 		fmt.Printf("%s", WordWrap(err.Error(), "\t\t", 5))
 	}
 	fmt.Printf("------------------------------------\n")
-	for _, q := range summary.Queries {
-		fmt.Printf("%s, Severity: %s, Results: %d\n", q.QueryName, q.Severity, len(q.Files))
-
+	for _, q := range sortBySev(summary.Queries) {
+		fmt.Printf("%s, Severity: %s, Results: %d\n", printer.PrintBySev(q.QueryName,
+			string(q.Severity)), printer.PrintBySev(string(q.Severity), string(q.Severity)), len(q.Files))
+		if !printer.minVersion {
+			fmt.Printf("Description: %s\n", q.Description)
+			fmt.Printf("Platform: %s\n\n", q.Platform)
+		}
 		for i := 0; i < len(q.Files); i++ {
-			fmt.Printf("\t%s:%d\n", q.Files[i].FileName, q.Files[i].Line)
+			fmt.Printf("\t%s %s:%s\n", printer.PrintBySev(fmt.Sprintf("[%d]:", i+1), string(q.Severity)),
+				q.Files[i].FileName, printer.Success.Sprint(q.Files[i].Line))
+			if !printer.minVersion {
+				fmt.Println()
+				for idx, lines := range q.Files[i].VulnLines.Lines {
+					if q.Files[i].VulnLines.Positions[idx] == q.Files[i].Line {
+						printer.Line.Printf("\t\t%03d: %s\n", q.Files[i].VulnLines.Positions[idx], lines)
+					} else {
+						fmt.Printf("\t\t%03d: %s\n", q.Files[i].VulnLines.Positions[idx], lines)
+					}
+				}
+				fmt.Print("\n\n")
+			}
 		}
 	}
 	fmt.Printf("\nResults Summary:\n")
-	fmt.Printf("HIGH: %d\n", summary.SeveritySummary.SeverityCounters["HIGH"])
-	fmt.Printf("MEDIUM: %d\n", summary.SeveritySummary.SeverityCounters["MEDIUM"])
-	fmt.Printf("LOW: %d\n", summary.SeveritySummary.SeverityCounters["LOW"])
-	fmt.Printf("INFO: %d\n", summary.SeveritySummary.SeverityCounters["INFO"])
+	fmt.Printf("%s: %d\n", printer.High.Sprint("HIGH"), summary.SeveritySummary.SeverityCounters["HIGH"])
+	fmt.Printf("%s: %d\n", printer.Medium.Sprint("MEDIUM"), summary.SeveritySummary.SeverityCounters["MEDIUM"])
+	fmt.Printf("%s: %d\n", printer.Low.Sprint("LOW"), summary.SeveritySummary.SeverityCounters["LOW"])
+	fmt.Printf("%s: %d\n", printer.Info.Sprint("INFO"), summary.SeveritySummary.SeverityCounters["INFO"])
```

Comment:
Create a constants file, that can be inside internal folder. Store the values of "HIGH", "MEDIUM", "LOW" and "INFO". Replace all string references of these words, to the new constants file

Reply:
resolved